tests/deployment/ldap/values.yaml

```diff
@@ -1,6 +1,71 @@
-adminPassword: "admin"
+# Default values for openldap.
```

Comment:
This file with values should be stored with ldap subchart, only values used in test should be stored under /tests/deployment. I think you can delete folder under tests/deployment directory and only leave file with test ldap values.